src/frontend/web_application/src/scenes/Contact/components/PublicKeyForm/presenter.jsx

```diff
@@ -0,0 +1,145 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Field } from 'redux-form';
+import { withI18n, Trans } from '@lingui/react';
+import renderReduxField from '../../../../services/renderReduxField';
+import { Button, Confirm, FieldErrors, Fieldset, FormColumn, FormGrid, FormRow, Icon, Legend, TextFieldGroup as TextFieldGroupBase, TextareaFieldGroup as TextareaFieldGroupBase } from '../../../../components';
+
+import './style.scss';
+
+const TextFieldGroup = renderReduxField(TextFieldGroupBase);
+const TextareaFieldGroup = renderReduxField(TextareaFieldGroupBase);
+
+@withI18n()
+class PublicKeyForm extends PureComponent {
+  static propTypes = {
+    errors: PropTypes.arrayOf(PropTypes.string),
+    i18n: PropTypes.shape({}).isRequired,
+    contactId: PropTypes.string.isRequired,
+    publicKey: PropTypes.shape({}),
+    handleSubmit: PropTypes.func.isRequired,
```

Comment:
handleSubmit or onSubmit ?

my notation rule is:
- if it is a component prop: `onWhatever`
- if it is a class method: `handleWhatever`

for example:

```
class Foo extends Component {
  handleBleep = ({ bleep }) => { 
    // do things with bleep
  }
  
  render() {
    return <Bar onBleep={this.handleBleep} />
  }
}
```